tests/parsing/FilterHelperTest.py

```diff
@@ -0,0 +1,28 @@
+import unittest
+
+from coalib.parsing.FilterHelper import (
+    apply_filter,
+    apply_filters,
+    available_filters,
```

Comment:
I would suggest changing it to `from coalib.parsing.filters import available_filters`. Honestly, I don't know which one is better.

Reply:
c.f. https://github.com/coala/coala/issues/5331